### DISCUSSION TITLE
Pre Log_In HandShake

### DISCUSSION THREAD

**Discussion by MUhammadasadnazar** [dotnet/SqlClient#1592] [General]
Hi , I am trying to connect my  .NET MAUI Project with MS SQL but I Am Not Able  , While using System.data.sqlclient    SqlConnection is not available and when I am using Microsoft.data.sqlclient it is giving me exception 

" Aconnection was successfully established with the server but then an error occurred during the pre login handshake (Provider TCP Provider , error :35 A internal exception was caught"  

So any one can help me please 

Regards : M.Asad

**Comment by johnnypham** [comment]
Looks like the same error as #1412.
